Documentation/Reference/Events/application-startup.md

```diff
@@ -63,7 +63,7 @@ If you want more control over execution you can override these properties:
 
 Here's an example that does the same thing as the ApplicationStartupHandler sample below. Remember that it's an interface so you have to implement all three methods, but you can leave the ones you're not using empty.
 
-	using Umbraco.Core;
+	using umbraco.Core;
```

Comment:
My local branch has this:

    using Umbraco.Core;
    using umbraco.BusinessLogic;
    using umbraco.cms.businesslogic;
    using umbraco.cms.businesslogic.web;

Should each begin with lower-case u?